server/services/workAndSkillsPageService.ts

```diff
@@ -60,31 +64,34 @@ export default class WorkAndSkillsPageService {
       learnerProfiles,
       learnerEducation,
       learnerLatestAssessments,
-      learnerGoals,
+      curiousGoals,
```

Comment:
As noted previously, I've renamed "learner goals" to "curious goals" to better reflect the type and source of those goals, especially now we are introducing goals from PLP